dev/com.ibm.ws.security.utility_fat_one/fat/src/com/ibm/ws/security/utility/test/SecurityUtilityCreateLTPAKeysTest.java

```diff
@@ -0,0 +1,706 @@
+/*******************************************************************************
+ * Copyright (c) 2025 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ *******************************************************************************/
+
+package com.ibm.ws.security.utility.test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import java.security.SecureRandom;
+import java.util.Base64;
+import java.util.Properties;
+import java.util.regex.Matcher;
+import java.nio.file.Paths;
+import java.util.regex.Pattern;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
+
+import com.ibm.websphere.simplicity.Machine;
+import com.ibm.websphere.simplicity.ProgramOutput;
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.impl.LibertyServerFactory;
+
+/**
+ * Test class for the securityUtility createLTPAKeys command.
+ * 
+ * This test suite validates the functionality of the createLTPAKeys command, including:
+ * - Creating LTPA key files in various locations (local, server)
+ * - Testing command options (password, passwordKey, passwordEncoding)
+ * - Validating error handling for invalid inputs
+ * - Verifying server configuration and LTPA initialization
+ */
```

Comment:
Add the following annotation to all the test classes in this package except `FATSuite.java`
```java
@RunWith(FATRunner.class)
```

Among other things, this allows you to filter tests run with FULL and LITE mode, but it also allows you to use stuff like `-Dfat.test.class.name=com.ibm.ws.security.utility.test.SecurityUtilityCreateLTPAKeysTest` when running tests locally with `buildandrun`